R/hydromad.R

```diff
@@ -134,20 +369,26 @@ logLik.hydromad <-
     val
   }
 
+
+#' @rdname hydromad.object
+#' @export
 deviance.hydromad <- stats:::deviance.lm
 
+
+#' @rdname hydromad.object
+#' @export
 print.hydromad <-
   function(x, digits = max(3, getOption("digits") - 3), ...) {
     cat("\n",
-      "Hydromad model with ", toString(deparse(x$sma)), " SMA",
-      " and ", toString(deparse(x$routing)), " routing:", "\n",
-      sep = ""
+        "Hydromad model with ", toString(deparse(x$sma)), " SMA",
+        " and ", toString(deparse(x$routing)), " routing:", "\n",
+        sep = ""
     )
     rx <- x$data
-    cat("Start = ", index2char(index(rx)[1], frequency(rx)),
-      ", End = ", index2char(index(rx)[NROW(rx)], frequency(rx)),
-      "\n",
-      sep = ""
+    cat("Start = ", zoo::index2char(index(rx)[1], frequency(rx)),
```

Comment:
```suggestion
    cat("Start = ", index2char(index(rx)[1], frequency(rx)),
```

Reply:
Same as above for this. I think what's confusing is just because a package is in Depends doesn't mean all its functions can automatically be used ... you still have to specify where you're calling external functions from.

Reply:
Happy to leave this too.
I would have thought this was covered by an importFrom annotation somewhere in the code base, which ends up in the NAMESPACE. And there's plenty of other references to zoo that don't need the zoo::???

Reply:
I think you're right, `@importFrom zoo index2char` should work. I'll test it and change if it works.